content/influxdb/cloud/reference/internals/security.md

```diff
@@ -0,0 +1,241 @@
+
+To protect data, InfluxDB Cloud includes the following:
+
+- Guaranteed [tenant isolation](#tenant-isolation) and [data integrity](#data-integrity).
+- Trusted cloud infrastructure
+  including [Amazon Web Services (AWS)](#amazon-web-services-aws),
+  [Microsoft Azure](#microsoft-azure),
+  and [Google Cloud Platform (GCP)](#google-cloud-platform-gcp),
+  building on [security controls of these cloud providers](#public-cloud-security),
+  such as physical security, disk encryption, and key management services (KMS).
+- Comprehensive [application and service security](#application-and-service-security)
+  covering both technical security measures and the people and processes that maintain the platform, including:
+  - [Internal access controls](#internal-access-controls)
+  - [Configuration management](#configuration-management)
+  - [Secure software development life cycle (SDLC)](#secure-software-development-life-cycle-sdlc)
+  - [Separation of environments and duties](#separation-of-environments-and-duties)
+  - [Monitoring, logging, and alerting](#monitoring-logging-and-alerting)
+  - [Third-party security assessments](#security-assessments)
+  - [Business continuity and disaster recovery](#business-continuity-and-disaster-recovery)
+  - [Data durability](#data-durability)
+  - [Incident response](#incident-response)
+- [Configurable security controls](#configurable-security-controls):
+  - [Authentication, authorization, and accounting](#access-authentication-and-authorization)
+  - [Role-based access controls (RBAC)](#role-based-access-controls-rbac)
+  - [Advanced controls](#advanced-controls)
+- [Compliance auditing](#compliance-and-auditing)
+  to ensure continuous review and adoption of industry-recognized standards and best practices.
+
+## Tenant isolation
+
+In the InfluxDB Cloud multi-tenant platform, access controls ensure that only valid authenticated and authorized requests access your account data.
+Access control includes:
+
+- A unique organization ID is assigned to an account when the account is created.
+  All internal Cloud services require this organization ID to authenticate entities before accessing or operating on data.
+- All external requests must be authorized with a valid token or session.
+  This is enforced by every InfluxDB Cloud service, from individual API calls to UI interactions.
+
+## Data integrity
+
+A dedicated internal service ensures data integrity
+by periodically creating, recording, and writing test data into test buckets.
+The service then periodically executes queries to ensure the data hasn't been lost or corrupted.
+A separate instance of this service lives within each cluster.
+Additionally, the service creates out-of-band backups in [line protocol](/influxdb/cloud/reference/syntax/line-protocol/),
+and ensures the backup data matches the data on disk.
+
+## Cloud infrastructure
+
+{{< img-hd src="/img/cloud-internals/cluster.png" alt="InfluxDB Cloud cluster architecture" />}}
```

Comment:
Update image file path to use [image naming conventions](/influxdata/docs-v2/blob/master/CONTRIBUTING.md#image-naming-conventions):

```
/img/influxdb/cloud-internals-cluster.png
```